<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MsgCenterItem</class>
 <widget class="QWidget" name="MsgCenterItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>76</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>76</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>260</width>
    <height>76</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MsgCenterItem</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#MsgCenterItem{background:#F6F7FF;}
QWidget#MsgCenterItem:hover{background:#FFFFFF;}
QWidget#MsgCenterItem:pressed{background:#FFFFFF;}</string>
  </property>
  <widget class="QLabel" name="labelIcon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/ZcloudMsgCenter/image/sys_msg.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>19</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolTip{background:rgba(255,255,255,1);border:1px solid rgba(153, 153, 153, 1);border-radius:0px}

QLabel#labelTitle{
background-color:rgba(255,255,255,0);
border:0px;
font:14px &quot;微软雅黑&quot;;
font-weight:400;
color:#333333;
}:</string>
   </property>
   <property name="text">
    <string>系统消息</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTime">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>19</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>16</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
border:0px;
font: 12px &quot;微软雅黑&quot;;
color:#999999;</string>
   </property>
   <property name="text">
    <string>18/03/21</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="delMsgButton">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>43</y>
     <width>12</width>
     <height>14</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>12</width>
     <height>14</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>12</width>
     <height>14</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-bottom-left-radius:0px;border-image: url(:/ZcloudMsgCenter/image/del_msg.png);}
QPushButton:hover,pressed{border-image: url(:/ZcloudMsgCenter/image/del_msg_sel.png);};

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelContent">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>43</y>
     <width>180</width>
     <height>14</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>14</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>180</width>
     <height>14</height>
    </size>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolTip{background:rgba(255,255,255,1);border:1px solid rgba(153, 153, 153, 1);border-radius:0px}

QLabel#labelContent
{background-color:rgba(255,255,255,0);
border:0px;
font:12px &quot;微软雅黑&quot;;
font-weight:400;
color:#999999;
}</string>
   </property>
   <property name="text">
    <string>我是内容</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ZcloudMsgCenter.qrc"/>
 </resources>
 <connections/>
</ui>
